Update unfinished pages management (#275)
pharo-open-documentation/pharo-wiki: CONTRIBUTION.md

Once the page is created you need to add it to the [README.md](README.md) in the right section.

> > Note: If you have contents to start a page but don't have the time nor the knowledge to finish it, it is advised to propose a pull request with the content you have and to open issues with the missing sections pointing to the concerned page.

Reviewer: ```suggestion
> Note: If you have contents to start a page but don't have the time or the knowledge to finish it, it is advised to propose a pull request with the content you have already. Then, one can open issues with the missing sections pointing to the concerned page.
```

Author: I had the impression that `nor` warked in English but I trust you :P

Reviewer: The idea is that one can not have the time but have the knowledge or the other way around.
Nor implies that one does not have the time **and** one does not have the knowledge.

Well, if I understand nor correctly :-)
